internal/sbi/producer/data_repository.go

```diff
@@ -1535,6 +1556,12 @@ func PolicyDataUesUeIdSmDataPatchProcedure(collName string, ueId string,
 			logger.DataRepoLog.Warnln(err)
 		}
 
+		for sstSd, snssaiData := range smPolicyData.SmPolicySnssaiData {
```

Comment:
use a new map to add, and replace smPolicyData.SmPolicySnssaiData with the new map